src/Compilers/CSharp/Test/Emit/Emit/InAttributeModifierTests.cs

```diff
@@ -4405,6 +4405,85 @@ public class Test
                 Diagnostic(ErrorCode.ERR_PredefinedTypeNotFound, "ref readonly int").WithArguments("System.Runtime.InteropServices.InAttribute").WithLocation(4, 12));
         }
 
+        [Fact]
+        public void ParentClassOfProxiedInterfaceFunctionHasNoModreq_ImplementedInChild()
+        {
+            var code = @"
+class Parent
+{
+    public void M(in int x) { }
+}
+interface IM
+{
+    void M(in int x);
+}
+class Child: Parent, IM
+{
+    public void M(in int x) { }
+}";
```

Comment:
It looks `Child.M` should generate a warning that `new` is required. Consider calling `VerifyDiagnostics` so it's clear that we generate the warning in this case.